src/collar/oc_anim.lsl

```diff
@@ -52,38 +52,42 @@ integer CMD_SAFEWORD = 510;
 integer EXT_CMD_COLLAR = 499; // Added for collar or cuff commands to put ao to pause or standOff
 integer ATTACHMENT_RESPONSE = 601;
 
-integer NOTIFY = 1002;
+integer NOTIFY  =  1002;
 integer LOADPIN = -1904;
-integer REBOOT = -1000;
+integer REBOOT  = -1000;
+
```

Comment:
Could we stick to just one space on either side of the equals sign?  While I agree that right-aligning them looks prettier, it's more hassle to maintain unless everyone's editor does it automatically (and I don't know of an LSL editor that will).